python/brayns/async_client.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2016-2018, Blue Brain Project
+#                          Raphael Dumusc <raphael.dumusc@epfl.ch>
+#                          Daniel Nachbaur <daniel.nachbaur@epfl.ch>
+#                          Cyrille Favreau <cyrille.favreau@epfl.ch>
+#
+# This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+#
+# This library is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License version 3.0 as published
+# by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+# All rights reserved. Do not distribute without further notice.
+
+"""Client that connects to a remote running Brayns instance which provides the supported API."""
+
+import asyncio
+import rockets
+
+from .api_generator import build_api
+from .base import BaseClient
+from .utils import obtain_registry, convert_snapshot_response_to_PIL, SCHEMA_ENDPOINT
+from . import utils
```

Comment:
Why not move what you have at `.` in `utils`?

Reply:
Actually I see that it is there, so why import that one differently? Why not:
```py
from .utils import obtain_registry, convert_snapshot_response_to_PIL, SCHEMA_ENDPOINT, in_notebook
```

Reply:
That's a trap! The mocking otherwise does not work if the method is not used via its module name.

Reply:
Got it 😄